content/docs/tutorials/aws/ec2-webserver.md

```diff
@@ -55,66 +55,85 @@ aws:region: (us-east-1)
 
     ```javascript
     const aws = require("@pulumi/aws");
-
+    
     let size = "t2.micro";     // t2.micro is available in the AWS free tier
-    let ami  = "ami-0ff8a91507f77f867"; // AMI for Amazon Linux in us-east-1 (Virginia)
```

Comment:
Given we no longer require us-east-1, can probably remove the comment above all this in the prose about making sure you are in us-east-1 as well.